feature-flags/schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "https://accounts.firefox.com/feature-flags.json",
+  "type": "object",
+  "properties": {
+    "blee": {
```

Comment:
`blee` and `wibble` seem like testing properties? Did you mean to include these?

Reply:
Just being lazy sorry, I promise I'll remove them before merging. It's much easier to test the content server changes using these than it is using `communicationPrefLanguages` or whatever.